### PR TITLE
Badge component

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
Badge:
1. image badge.
2. badge with text.

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+.badge-size-sm {
+    width: 10px;
+    height: 10px;
+}
+
+.badge-position-lg {
+    bottom: 4px;
+    right: 12px;
```
Comment (varun-Tad) [inline components/badge/badge.css:47]: use rem here for 'right' just like you used rems everywhere. Otherwise, the code looks good for me to merge

Comment (yaswanthmyneni) [reply components/badge/badge.css:47]: someone told me to use px units when the measurement is less than 1rem. should I use rems even it is less than 1rem? 
Asking this to know which is better practice.

**Comment by rahulk31** [comment]
Looks good to me, can be merged.

File: components/badge/badge.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        </div>
+        <!-- badge-size-lg with text -->
+        <div class="display-badge pos-rel-badge">
+            <i class="fas fa-shopping-cart icon-badge"></i>
+            <span class="badge badge-online badge-size-lg flex-badge badge-pos-to-icon">99</span>
+        </div>
+        <!-- badge-size-md with text -->
+        <div class="display-badge pos-rel-badge">
+            <i class="fas fa-shopping-cart icon-badge"></i>
+            <span class="badge badge-online badge-size-md flex-badge badge-pos-to-icon">9</span>
```
Comment (AshutoshM29) [inline components/badge/badge.html:56]: class name like "badge-pos-to-icon" is vague, try changing it.
Rest looks good

**Pull request merged**